definitions/variable/tag_electricity_input_types.yaml

```diff
@@ -243,31 +246,9 @@
       description: hydro, wind, solar, geothermal, ocean, and other renewable sources
         excluding bioenergy
 
-  - Hydro:
-      description: hydropower
-
-  - Hydro|Run of River:
-      description: run-of-river hydro power plants (i.e. without storage capacity)
-
-  - Hydro|Pumped Storage:
-      description: hydropower systems with short-term storage capacity
-        (including pumping turbines)
-      notes: This hydropower system has two water reservoirs at different elevations.
-        The definition follows the convention established
-        by the ENTSO-E transparency platform (https://transparency.entsoe.eu).
-
-  - Hydro|Reservoir:
-      description: hydropower systems with seasonal storage capacity
-        (possibly including pumping turbines)
-      notes: This definition follows the convention established
-        by the ENTSO-E transparency platform (https://transparency.entsoe.eu).
-
```

Comment:
@erikfilias all the Hydro technos should be kept here (Hydro, Hydro|Reservoir and Hydro|Pumped Storage or we will miss some important variables such as Secondary Energy|Hydro|Reservoir 
(as the tag electricity storage for instance should not be used for secondary energy as secondary energy is only for assets which generate electricity)
Then I think we cannot avoid to have some duplicate assets which will be both in tag storage and in tag input

Reply:
Yeah, I agree, but I am unsure if we can have duplicate assets. What do you think @danielhuppmann?

Reply:
I agree with @sandrinecharousset, there is "new" electricity generated from reservoir hydro (not pumped), so this has to remain part of the tag "Electricity Input".

Reply:
@erikfilias, please reinsert all hydro electricity generation types.